src/response.rs

```diff
@@ -627,6 +613,41 @@ impl Response {
         Ok(response)
     }
 
+    /// Create a Response from a Read trait impl.
+    ///
+    /// This is hopefully useful for unit tests.
+    ///
+    /// Example:
+    ///
+    /// use std::io::Cursor;
+    ///
+    /// let text = "HTTP/1.1 401 Authorization Required\r\n\r\nPlease log in\n";
+    /// let read = Cursor::new(text.to_string().into_bytes());
+    /// let resp = ureq::Response::do_from_read(read);
+    ///
+    /// assert_eq!(resp.status(), 401);
```

Comment:
This doc is just "lol" at this point (it used to make sense). Not your fault, but moving it around seems a bit much :D